docker-compose.prod.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.4"
+
+services:
+  cron-manager:
+    build:
+      dockerfile: "docker/Dockerfile.prod"
+      context: .
+    env_file:
+      - .env
+    volumes:
+      - ./.env:/usr/src/app/.env
```

Comment:
is this `.env` needed for the application? I believe, we use `config.yml` in this context